main.go

```diff
@@ -130,18 +126,20 @@ func TCPReceiver(done <-chan struct{}, af string, targetAddr string, probePortSt
 	// IP + TCP header, this channel is fed from the socket
 	recv := make(chan TCPResponse)
 	go func() {
-		const tcpHdrSize int = 20
-		packet := make([]byte, ipHdrSize+tcpHdrSize)
+		ipHdrSize := 0
```

Comment:
Comment about why this is OK to be 0 for ip6 (moving the previous comment is also acceptable :) )